eclipse-theia/theia
# Support Webview Views

Added API to support Webview based views contributions

Signed-off-by: ChayaLau <chaya.lau@sap.com>



#### What it does

This commit adds support to install and use webview views extensions - to be shows in the views - side panel
as described in the issue:
https://github.com/eclipse-theia/theia/issues/8740
#### How to test

install the VSCode webview-view-sample extension.
go to the explorer side panel menu and click the button
the list of views should include the `calico colors` title   
expand the view -> expected results - the webview should appear presenting the webview view contribution. 
#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
